lib/generators/graphiti/api_test_generator.rb

```diff
@@ -26,33 +26,33 @@ def var
     end
 
     def dir
-      @resource.gsub('Resource', '').underscore.pluralize
+      @resource.gsub("Resource", "").underscore.pluralize
     end
 
     def generate_api_specs
-      if actions?('index')
+      if actions?("index")
```

Comment:
surprising to me, I know rubocop wants the opposite

Reply:
From Standard readme:

> Double quotes for string literals - because pre-committing to whether you'll need interpolation in a string slows people down

Makes sense to me.  Always annoying when needing to add interpolation to also need to change out quotes.